services/api-server/tests/unit/pact_broker/conftest.py

```diff
@@ -0,0 +1,61 @@
+import os
+import socket
+
+import pytest
+
+
+def pytest_addoption(parser: pytest.Parser) -> None:
+    group = parser.getgroup(
+        "oSparc e2e options", description="oSPARC-e2e specific parameters"
```

Comment:
This group name looks like a copy&paste from the e2e. Lets call it `pact options` isntead of `"oSparc e2e options"` and pls update the description 

to see how it feels, just `pytest --help` and see how you are getting these options